doc/source/index.rst

```diff
@@ -0,0 +1,21 @@
+.. pandas-mentoring documentation master file, created by
+   sphinx-quickstart on Tue Jul 30 15:08:15 2019.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to pandas-mentoring's documentation!
+============================================
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Contents:
+
+
+
+
```

Comment:
in big projects, at least in pandas, we try to avoid non-deterministic things. Meaning that we try to set standards for things like how many blank lines we should leave between sections (similar to what pep8 does too). Surely not important at this point, but just letting you know.

Reply:
I'll leave just one empty line